src/chain.h

```diff
@@ -182,7 +182,7 @@ class CBlockIndex
     //!
     //! Note: this value is modified to show BLOCK_OPT_WITNESS during UTXO snapshot
     //! load to avoid the block index being spuriously rewound.
-    //! @sa RewindBlockIndex
+    //! @sa NeedsRedownload
```

Comment:
For the sake of completeness, context for the change is [here](https://github.com/bitcoin/bitcoin/pull/21816).